src/components/services/services.module.scss

```diff
@@ -0,0 +1,25 @@
+.services {
+  @apply container mt-24 flex flex-col gap-9;
+
+  @screen md {
+    @apply mt-36 gap-16;
+  }
+  @screen lg {
+    @apply mt-40;
+  }
+
+  &__item {
+    @apply flex-grow border-2 border-primary-100 p-8;
+
+    @screen md {
+      @apply p-10;
+    }
+  }
+}
+.services_item__header {
+  @apply mb-4 text-3xl font-normal leading-10;
+
+  @screen md {
+    @apply text-4xl leading-snug;
+  }
+}
```

Comment:
Again, you need to use a nested classes according to BEM. The main class is "services" and nested class is "item" or "item-header".